Missionframework/modules/00_init/fnc/fn_init_configGuard.sqf

```diff
@@ -51,20 +56,19 @@ KPLIB_validationNamespace setVariable ["KPLIB_preset_checkedSingles", true];
         "KPLIB_configError" call BIS_fnc_endMissionServer;
     };
 
-    diag_log format ["[KP LIBERATION] [%1] [CFG GUARD] Configuration files are valid", diag_tickTime];
+    diag_log text format ["[KP LIBERATION] [%1] [CFG GUARD] Configuration files are valid", diag_tickTime];
 
-    diag_log format [
-        "[KP LIBERATION] [MISSIONSTART] Missionfile: %1 - World Name: %2 - Version: %3 - Blufor: %4 - Opfor: %5 - Resistance: %6 - Civilians: %7 - Arsenal: %8 - ACE: %9",
-        (localize "STR_KPLIB_TITLE"),
-        worldName,
-        (localize "STR_KPLIB_VERSION"),
-        KPLIB_param_presetF,
-        KPLIB_param_presetE,
-        KPLIB_param_presetR,
-        KPLIB_param_presetC,
-        KPLIB_param_presetArsenal,
-        KPLIB_ace_enabled
-    ];
+    diag_log text "[KP LIBERATION] [MISSIONSTART] --- MISSION INFORMATION ---";
+    diag_log text format ["[KP LIBERATION] [MISSIONSTART] Name: %1", (localize "STR_KPLIB_TITLE")];
+    diag_log text format ["[KP LIBERATION] [MISSIONSTART] World: %1", worldName];
+    diag_log text format ["[KP LIBERATION] [MISSIONSTART] Version: %1", (localize "STR_KPLIB_VERSION")];
+    diag_log text format ["[KP LIBERATION] [MISSIONSTART] Author: %1", [missionConfigFile] call BIS_fnc_overviewAuthor];
+    diag_log text format ["[KP LIBERATION] [MISSIONSTART] Blufor: %1", KPLIB_preset_nameF];
+    diag_log text format ["[KP LIBERATION] [MISSIONSTART] Opfor: %1", KPLIB_preset_nameE];
+    diag_log text format ["[KP LIBERATION] [MISSIONSTART] Resistance: %1", KPLIB_preset_nameR];
+    diag_log text format ["[KP LIBERATION] [MISSIONSTART] Civilians: %1", KPLIB_preset_nameC];
+    diag_log text format ["[KP LIBERATION] [MISSIONSTART] Arsenal: %1", KPLIB_param_presetArsenal];
```

Comment:
Maybe replace it with
```sqf
    diag_log text format ["[KP LIBERATION] [MISSIONSTART] Arsenal: %1", 
        [
            localize "STR_KPLIB_SETTINGS_ARSENAL_ARSENAL_0",
            localize "STR_KPLIB_SETTINGS_ARSENAL_ARSENAL_1",
            localize "STR_KPLIB_SETTINGS_ARSENAL_ARSENAL_2"
        ] select KPLIB_param_presetArsenal
    ];
```

It should for example print `Unrestricted` instead of `0`
(Not tested)

Reply:
Oh, of course. Didn't remember that you've a localization for the Arsenal selection. 👍